[2025-05-28T10:14:39.748+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T10:14:39.761+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T10:14:39.762+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T10:14:39.778+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_api_task> on 2025-05-27 00:00:00+00:00
[2025-05-28T10:14:39.785+0000] {standard_task_runner.py:60} INFO - Started process 258 to run task
[2025-05-28T10:14:39.788+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_api_to_sqlite', 'load_api_task', 'scheduled__2025-05-27T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_sqlite.py', '--cfg-path', '/tmp/tmp6l0v65xo']
[2025-05-28T10:14:39.790+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask load_api_task
[2025-05-28T10:14:39.841+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-05-27T00:00:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T10:14:39.910+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_api_to_sqlite' AIRFLOW_CTX_TASK_ID='load_api_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-27T00:00:00+00:00'
[2025-05-28T10:14:40.172+0000] {python.py:201} INFO - Done. Returned value was: API data loaded successfully to SQLite. 70 records inserted in /opt/***/data/etl_database.db
[2025-05-28T10:14:40.206+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_api_to_sqlite, task_id=load_api_task, execution_date=20250527T000000, start_date=20250528T101439, end_date=20250528T101440
[2025-05-28T10:14:40.243+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T10:14:40.260+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
